doc/batteries_and_electricity.md

```diff
@@ -0,0 +1,15 @@
+The way that batteries, power storage, and electricity use are handled is a pile of compromises, so this document outlines the decisions about these topics and some of the rationales for them.
+
+## Voltage, Current, Capacity
+The short of it is, we are not handling voltage and current explicitly, only capacity.
+As we are not designing systems but rather representing them, it's reasonable to assume that a particular class of battery is generally fit for purpose.
+For example, we assume that battery cells for handheld devices are able to activate those handheld devices, even though in reality some devices require higher current than average batteries will provide.  Once you scale up to e.g. electric vehicle battery packs, we assume that a car-scale battery pack is fit for purpose, even though in reality moving a battery from one electric vehicle to another would likely be an extremely difficult endeavor.
+There is some possiblility of making different batteries or other power sources not fit for purpose for certain tasks, but this would be realized via some labelling like "high-draw power source" rather than attempting to model voltage and current limits explicitly.
+
+## Battery representation
+Matching batteries to a particular electrical device can be a pain, but is it a fun or interesting kind of pain? I assert not, and as such we are not handling a wide variety of different battery types and sizes that have to be matched to the device, but abstracting battery packs to a near-minimal representative set of size classes to cover all our devices.
+Likewise, many electronic devices accept some number of standard sized battery cells, but this isn't a particularly interesting detail, so instead we are abstracting battery compartments to holding a single battery cell of an appropriate capacity and not handling multiple cells per device.
```

Comment:
Without a real example, lets assume the situation i want to add a thing, that irl uses three 18650 batteries (aka MEDIUM). I can't use single medium because it's 1/3 of what it needs, and i can't use HEAVY because nowadays it's a "tool battery", and what i want to add is not a tool (at least not one that uses this type of batteries) (and also it's twice as big as i need). Is intended way for me to make a new type of battery, that matches power of three 18650, and give it to this tool?

Reply:
Not precisely 3x 18650s, but I do believe we have a gap in that area that needs a new battery.

Reply:
But what battery? From how it looks for me, a lot of varying size batteries are very much 18650 taped together with a battery controller to pull energy evenly

Reply:
Just call it a battery pack and clarify it represents 3x or 4x 18650s
Abstractions break down around the edges, this is expected.

Reply:
It's a pretty crude solution, but until someone figures out multimag, having craft-able battery packs is probably one of the better solutions

Reply:
That's kinda the question: do we indeed prefer making a new battery type instead of ability to shove multiple batteries at once? After all i suspect the latter would be used pretty rarely

Reply:
I'm not saying make it explicitly a craftable battery pack, I'm saying base the properties on a pack of 3-4 AA or 18650 batteries, the description should just describe the size and typical uses.